website/docs/cookbooks/test-only.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Filter the test suites to run
+sidebar_position: 11
+---
+
+The `test-only` parameter it is a useful option for when you only want to run a specific subset of tests.
```

Comment:
I'd even be tempted to always call `test-only` an option here, since we only use it from the command line (and not through a `using` directive).
In which case:
```suggestion
The `--test-only` option is useful for when you only want to run a specific subset of tests.
```